cli/tests/034_onload/imported.ts

```diff
@@ -1,8 +0,0 @@
-import "./nest_imported.ts";
```

Comment:
It would be better for review if you do not rename `cli/tests/034_onload` to `cli/tests/034_onload_onunload` - it results in a smaller diff.

Reply:
Sure, I'll revert it